subprojects/samples/src/integTest/groovy/org/gradle/integtests/samples/java/SamplesJavaTestingIntegrationTest.groovy

```diff
@@ -106,6 +106,8 @@ class SamplesJavaTestingIntegrationTest extends AbstractSampleIntegrationTest {
         given:
         TestFile dslDir = sample.dir.file(dsl)
         executer.inDirectory(dslDir)
+        // To workaround failure with Kotlin 1.8.20
```

Comment:
A comment should link to the root cause issue at https://youtrack.jetbrains.com/issue/KT-57757/Reduce-classpath-snapshotter-memory-consumption

Reply:
This should be removed now that we do the same for all windows embedded tests

Reply:
Done, dropped the commit.